registrations/api.py

```diff
@@ -356,6 +360,37 @@ def filter_queryset(self, queryset):
 register_view(SignUpViewSet, "signup")
 
 
+class SignUpGroupViewSet(
+    UserDataSourceAndOrganizationMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    viewsets.GenericViewSet,
+):
+    serializer_class = SignUpGroupSerializer
+    queryset = SignUpGroup.objects.all()
+    permission_classes = [
+        (CanCreateEditDeleteSignup & DataSourceResourceEditPermission)
```

Comment:
@jorilindell & @voneiden: I changed the permissions to be the same for signup groups and signups (something that wasn't noticed earlier). Perhaps check this once more for just the permissions and approve if OK